src/components/common/FormButton.js

```diff
@@ -443,7 +443,7 @@ const FormButton = ({
         disabled={disabled}
         onClick={(e) => {
             onClick && onClick(e)
-            linkTo && (linkTo.includes('http') ? window.open(linkTo, '_blank') : history.push(linkTo))
+            linkTo && (linkTo.toLowerCase().startsWith('http') ? window.open(linkTo, '_blank') : history.push(linkTo))
```

Comment:
This is needed, otherwise when we click the `Log In and Claim` button (whose `linkTo` contains the redirect url), a new tab will be opened. 

![image](https://user-images.githubusercontent.com/46699230/123989217-4609d400-d9fb-11eb-803f-1113cd93257e.png)

Reply:
I recently added the `_blank` so that external links open in a new tab. I tested this and it works fine - however, might want to separate internal and external link treatment as a followup on this: `linkTo` or `linkToExternal` vs doing this checking for `http` at the beginning of the `URL`.

Reply:
do we need to refactor the code with `linkToExternal` in this PR? 

I can do it, but I'm just not sure whether there're some special cases where we don't know whether the link is internal or external when passing the value to `FormButton`. Need to go through the code carefully.

Reply:
@think-in-universe I think it would be great if we did the refactor to use `linkToExternal` for any external links and `linkTo` for router links. All the current links passed to `linkTo` are clear whether they are external or not, but yes we just need to check it thoroughly.